client/src/app/primitives/DropdownButton.less

```diff
@@ -0,0 +1,61 @@
+:local(.DropdownButton) {

```

Comment:
Same comment like above applies here:

I'd create styles based on a single shared parent that is exposed via a `:local` declaration:

```less
:local(.DropdownContainer) {
   > .dropdown-button { ... }
   > .dropdown { ... }
}
```